Index: base/basictypes.h
===================================================================
--- base/basictypes.h	(revision 57015)
+++ base/basictypes.h	(working copy)
@@ -29,11 +29,14 @@
 
 // The NSPR system headers define 64-bit as |long| when possible.  In order to
 // not have typedef mismatches, we do the same on LP64.
+#ifndef _INT64
+#define _INT64
 #if __LP64__
 typedef long                int64;
 #else
 typedef long long           int64;
 #endif
+#endif
 
 // NOTE: unsigned types are DANGEROUS in loops and other arithmetical
 // places.  Use the signed types unless your variable represents a bit
@@ -51,11 +54,14 @@
 #endif
 
 // See the comment above about NSPR and 64-bit.
+#ifndef _UINT64
+#define _UINT64
 #if __LP64__
 typedef unsigned long uint64;
 #else
 typedef unsigned long long uint64;
 #endif
+#endif
 
 // A type to represent a Unicode code-point value. As of Unicode 4.0,
 // such values require up to 21 bits.
